data-hibernate-jpa/src/test/groovy/io/micronaut/data/hibernate/async/AsyncSpec.groovy

```diff
@@ -85,7 +85,7 @@ class AsyncSpec extends Specification {
         when:"An entity is not found"
         def notThere = asyncCrudRepository.findById(1000L)
                 .exceptionally({ Throwable t ->
-                    if (t instanceof EmptyResultException) {
+                    if (t instanceof EmptyResultException || t.getCause() instanceof EmptyResultException) {
```

Comment:
This now returns CompletionException vs directly EmptyResultException so for the test I have added check for cause as well. Didn't figure out why it is wrapped now in CompletionException